### PR TITLE
Organize the header file includes in C++ buffer

### PR DESCRIPTION
**Pull request by sei40kr**
Organize the header file includes using [cpp-auto-include](https://github.com/syohex/emacs-cpp-auto-include).

# New Key Binding

| Key binding | Description                                                   |
|:-------------|:---------------------------------------------------------------|
| `SPC m r i` | organize includes                                           |

### CODE REVIEW

**Comment by Compro-Prasad** [comment]
IMO `includes` is more appropriate than `inclusions`.

**Comment by sei40kr** [comment]
@Compro-Prasad 
I've fixed the PR. Since English is not my native tongue, such review is very helpful.

**Comment by Compro-Prasad** [comment]
Also reword the commit message.

**Comment by sei40kr** [comment]
@Compro-Prasad 
I've fixed the commit.

File: layers/+lang/c-c++/README.org [hunk 1]
```diff
@@ -283,6 +283,7 @@ generally applicable.
 |             | (e.g. switch between .cpp and .h, requires a project to work) |
 | ~SPC m D~   | disaster: disassemble c/c++ code                              |
 | ~SPC m r .~ | srefactor: refactor thing at point.                           |
+| ~SPC m r i~ | organize includes                                             |
```
Comment (Compro-Prasad) [inline layers/+lang/c-c++/README.org]: Give elisp to enable this feature in `.spacemacs`.

**Comment by Compro-Prasad** [comment]
This package isn't available on MELPA. How should I install it?

**Comment by duianto** [comment]
Thank you for contributing to Spacemacs!
The PR has been cherry-picked into the `develop` branch, you can safely delete your branch.
Commit: https://github.com/syl20bnr/spacemacs/commit/1e00461dd808c4d392ab36592cf7246c05d05a16

The `changelog.develop` entry was updated. It was moved from:
```org
*** Breaking Changes
**** Layers
***** C-C++
```
to:
```org
*** Layer changes and fixes
**** C-C++
```
A new sub section was added and the variable and key binding was listed as well:
```text
- New packages:
  - =cpp-auto-include= Insert and delete C++ header files automatically:
    - Added variable: =++-enable-organize-includes-on-save=
    - Added key binding: ~SPC m r i~ organize includes
```

**Pull request closed without merging**